ansible/inventories/production/group_vars/inventory-next/vars.yml

```diff
@@ -40,7 +40,7 @@ inventory_ckan_plugins_additional: [s3filestore saml2auth]
 # saml2 authentication
 inventory_ckan_saml2_enabled: true
 ckan_production_ini_template: etc/ckan/production.ini
```

Comment:
also this.
ckan_production_ini_template: inventory-next/etc_ckan_production.ini.j2

Let me make a PR.

Reply:
https://github.com/GSA/datagov-deploy/pull/2763